.github/workflows/flowzone.yml

```diff
@@ -990,41 +896,76 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
+          # fallback to the local image ref if we are not publishing anything
           images: |
-            ${{ env.DOCKER_IMAGES }}
+            ${{ env.DOCKER_IMAGES || env.SUT_IMAGE }}
           tags: |
             type=raw,value=${{ github.event.pull_request.head.sha }}
             type=raw,value=${{ github.event.pull_request.head.ref }}
+          labels: org.opencontainers.image.version=${{ needs.versioned_source.outputs.new_tag }}
           flavor: |
             latest=false
             prefix=${{ env.PREFIX }}
 
-      # this override file will add additional cache sources and inherit the docker-metadata-action
+      # this override file will add additional cache sources and inherit the docker-metadata-action target
       # https://docs.docker.com/build/customize/bake/file-definition/#json-definition
       - name: Create bake override
         run: |
-          jq -n '{target:{default:{}}}' > ${BAKE_EMPTY}
+          jq -n '{target:{"${{ matrix.target }}":{}}}' > ${BAKE_EMPTY}
           docker buildx bake --print ${{ matrix.target }} \
             -f ${{ join(fromJSON(needs.project_types.outputs.docker_bake),' -f ') || env.BAKE_EMPTY }} \
```

Comment:
@Page- Plain dockerfile support should be restored here. As long as there are docker images provided as an input, it should build and test them.

Though do you mean we should build even if not publishing or testing? Seems like a waste for repos that may be intended for balena push only?